dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/test/java/org/apache/dolphinscheduler/plugin/task/api/utils/ProcessUtilsTest.java

```diff
@@ -38,6 +38,7 @@ public void testGetPidsStr() throws Exception {
         String command;
         MockedStatic<OSUtils> osUtilsMockedStatic = Mockito.mockStatic(OSUtils.class);
         if (SystemUtils.IS_OS_MAC) {
+            pids = "-+= 6279 sudo -+- 6282 558_1497.sh --- 6354 sleep";
```

Comment:
It seems this will remove the previous test case?

Reply:
Yes, but this case is only for MacOS, the other os still uses the previous test case.